samples/javascript_nodejs/51.teams-messaging-extensions-action/bots/teamsMessagingExtensionsActionBot.js

```diff
@@ -120,47 +132,105 @@ function createCardCommand(context, action) {
         },
     };
 }
+function createCardCommand(action) {
+    // The user has chosen to create a card by choosing the 'Create Card' context menu command.
+    const data = action.data;
+    const heroCard = CardFactory.heroCard(data.title, data.text);
+    heroCard.content.subtitle = data.subTitle;
+    const attachment = { contentType: heroCard.contentType, content: heroCard.content, preview: heroCard };
+
+    return {
+        composeExtension: {
+            type: 'result',
+            attachmentLayout: 'list',
+            attachments: [
+                attachment
+            ]
+        }
+    };
+}
 
-function shareMessageCommand(context, action) {
+function shareMessageCommand(action) {
     // The user has chosen to share a message by choosing the 'Share Message' context menu command.
-    let userName = 'unknown';
-    if (action.messagePayload?.from?.user?.displayName) {
+    const userName = 'unknown';
+    if (action.messagePayload.from &&
+            action.messagePayload.from.user &&
+            action.messagePayload.from.user.displayName) {
         userName = action.messagePayload.from.user.displayName;
     }
 
     // This Messaging Extension example allows the user to check a box to include an image with the
     // shared message.  This demonstrates sending custom parameters along with the message payload.
-    let images = [];
+    const images = [];
```

Comment:
This should be `let` since you reassign it down below. Alternatively, you can keep it `const`, and then `images.push('http...')` below (preferred)